client/src/pages/Translations.jsx

```diff
@@ -1,14 +1,76 @@
 // @flow
 
 import React from 'react';
-import { Tabs, Layout, Row } from 'antd';
-
+import { Tabs, Layout, Row, Table, Tag } from 'antd';
+import {
+  ExclamationCircleOutlined,
+  CheckCircleOutlined,
+} from '@ant-design/icons';
 import '../css/Translations.css';
 
 const { Header } = Layout;
 const { TabPane } = Tabs;
 
 function Translations() {
+  const columns = [
+    {
+      title: 'Name',
+      dataIndex: 'name',
+      key: 'name',
+    },
+    {
+      title: 'Priority',
+      dataIndex: 'priority',
+      key: 'priority',
+    },
+    {
+      title: 'Status',
+      dataIndex: 'status',
+      key: 'status',
+    },
+    {
+      title: 'Translate',
+      key: 'translate',
+      dataIndex: 'translate',
+    },
+  ];
+
+  const data = [
+    {
+      key: '1',
+      name: 'New York No. 1 Lake Park',
+      priority: <Tag color="error">Urgent</Tag>,
+      status: (
+        <Tag icon={<CheckCircleOutlined />} color="success">
+          Verified
+        </Tag>
+      ),
+      translate: <a href="#/">Translate</a>,
```

Comment:
With these links, we can use react-router's `Link` component instead of `a` (see example [here](https://github.com/hack4impact-uiuc/ymca/blob/0369e54d2fd3ffd72fd516b7fd65a5e391229abe/client/src/components/ResourcePreview.jsx#L108))